src/components/search/SearchTypesDirectives.js

```diff
@@ -344,10 +343,10 @@ goog.require('ga_urlutils_service');
 
               }
               var ol3d = $scope.ol3d;
-              if (originToZoomLevel.hasOwnProperty(res.attrs.origin) &&
+              if (res.attrs.zoomlevel < 100 &&
```

Comment:
magic number, I know. We put real big number in index when we want bbox to be used. So it's kind of filter.